src/common/EventTrace.h

```diff
@@ -31,6 +31,7 @@
 #define OID_ELAPSED_WITH_MSG(m, elapsed, event, incl_oid) \
   EventTrace::trace_oid_elapsed(m, event, elapsed, __FILE__, __func__, __LINE__, incl_oid)
 #define FUNCTRACE() EventTrace _t1(g_ceph_context, __FILE__, __func__, __LINE__)
+#define FUNCTRACE_RBD(cct) EventTrace _t2(cct, __FILE__, __func__, __LINE__)
```

Comment:
This isn't really RBD-related. I think it would be clearer to update all existing uses of ```FUNCTRACE``` to require a ```CephContext*``` argument.

Reply:
Done. Thanks.